src/plugins/auto/async_infer_request.cpp

```diff
@@ -3,103 +3,34 @@
 //
 
 ///////////////////////////////////////////////////////////////////////////////////////////////////
-#include <string>
```

Comment:
@wgzintel @yangwang201911  @tiger100256-hu a initial version, just a heads up that we are reconstructing the code, do not review now, still under develop.